### DISCUSSION TITLE
[1.5.1] 'outputDirectory' specifies directory which doesn't exist.

### DISCUSSION THREAD

**Discussion by marceloverdijk** [jreleaser/jreleaser#1283] [Q&A] [answered]
Upgrading form JReleaser 1.3.1 to 1.5.1 using the Grade plugin I get:

```
* What went wrong:
A problem was found with the configuration of task ':jreleaserFullRelease' (type 'JReleaserFullReleaseTask').
  - In plugin 'org.jreleaser' type 'org.jreleaser.gradle.plugin.tasks.JReleaserFullReleaseTask' property 'outputDirectory' specifies directory '/home/runner/work/onlyf1-db/onlyf1-db/build/jreleaser' which doesn't exist.
    
    Reason: An input file was expected to be present but it doesn't exist.
    
    Possible solutions:
      1. Make sure the directory exists before the task is called.
      2. Make sure that the task which produces the directory is declared as an input.
    
    Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#input_file_does_not_exist for more details about this problem.

```

Note this is a different error that I got before:
[Gradle 8: property 'outputDirectory' has @Input annotation used on property of type 'DirectoryProperty'](https://github.com/jreleaser/jreleaser/issues/1212) which was resolved and I wanted to test with 1.5.1.

I also cannot explain why it is looking for directory '/home/runner/work/onlyf1-db/onlyf1-db/build/jreleaser'; the 'jreleaser' subfolder I have not set anywhere myself.

Not sure if this is misconfig from my side...

Ahh, while typing this I remember that the previous issue (#1212) I have mentioned above was related to combination of Gradle 8 and JReleaser 1.4.x. But what I actually now did is use 1.5.1 with Gradle 7. So not completely comparable by still the output directory issue is not what I would expect.

**Comment by aalmiray** [comment, marked as answer]
The `jreleaser` subfolder is registered by the plugin during configuration and it's supposed to be automatically created by it. It appears that's not the case.

**Reply by marceloverdijk** [reply]
Is it a bug or mis-config on my side?

**Reply by aalmiray** [reply]
I think it's a bug as the outputdirectory is supposed to be created as soon as the JReleaser logger is required which happens during configuration time.

**Reply by marceloverdijk** [reply]
ok, thx I created an issue.
